src/auth.ts

```
@@ -0,0 +1,42 @@
+          refreshToken: process.env.EMAIL_REFRESH_TOKEN,
+          accessToken: process.env.EMAIL_ACCESS_TOKEN,
+        },
+      },
+      from: process.env.EMAIL_FROM,
+    }),
+  ],
+
+  callbacks: {
+    ...authConfig.callbacks,
+    async jwt({ token, trigger, account, session }) {
+      if (trigger === "signUp" && account!.type === "email") {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)